collections/_article/dev-snapshot-godot-4-5-beta-1.md

```diff
@@ -0,0 +1,212 @@
+
+As mentioned in the new features section, abstract methods were a continuation of abstract classes. While the ability to create abstract classes was always available internally, it's thanks [Aaron Franke](https://github.com/aaronfranke) that GDScript users can take advantage of this this paradigm is now available to GDScript users ([GH-67777](https://github.com/godotengine/godot/pull/67777)). Now if a user wants to introduce their own abstract class, they merely need to declare it via the new `abstract` keyword.
+
+### GUI
+4.5 is bringing with it quite a few new quality-of-life improvements, with arguably the biggest addition being one we actually haven't given proper coverage to yet: foldable containers! [Tomasz Chabora](https://github.com/KoBeWi) is no stranger to editor enhancements, and this time he has blessed us with [GH-102346](https://github.com/godotengine/godot/pull/102346), which grants us the new `FoldableContainer` class. Now users can have dynamically cascading GUI objects with the ability to toggle if the contents are expanded or not at will, a process that previously took several workarounds to achieve.
+
+In a similar vein, the ability to manipulate a group of controls simultaneously has become much easier thanks to [Delsin-Yu](https://github.com/Delsin-Yu)'s PR with a focus on recursive control across child controls ([GH-97495](https://github.com/godotengine/godot/pull/97495)). This implements new proprties for recursively disabling `Focus Mode` and `Mouse Filter`, meaning that the ability to select & interact with child controls becomes far more intuitive, and allows for explicit overrides if desired.
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/godot-4-5-beta-1/recursive-focus.webm?1" type="video/webm">
+</video>
+
+There've also been a few improvements to `SVGTexture` in 4.5, such as the [dedicated importer](https://github.com/godotengine/godot/pull/105655). However, as it pertains to GUI, the biggest improvement would be [bruvzg](https://github.com/bruvzg)'s implementation of auto-scalable SVGs ([GH-105375](https://github.com/godotengine/godot/pull/105375)). This will allow users to explicitly oversample fonts and raster graphics, resulting a much crisper image overall.
```

Comment:
```suggestion
There have also been a few improvements to `SVGTexture` in 4.5, such as the [dedicated importer](https://github.com/godotengine/godot/pull/105655). However, as it pertains to GUI, the biggest improvement would be [bruvzg](https://github.com/bruvzg)'s implementation of auto-scalable SVGs ([GH-105375](https://github.com/godotengine/godot/pull/105375)). This will allow users to explicitly oversample fonts and raster graphics, resulting a much crisper image overall.
```